tools/testsys/Test.toml.example

```diff
@@ -0,0 +1,78 @@
+# This is an example testing configuration for testsys, the tool that
```

Comment:
For the lack of a better place for this comment, don't we need documentation about this file and how it will be used by `cargo make test`?